zipkin-receiver-kafka/src/main/scala/com/twitter/zipkin/receiver/kafka/KafkaStreamProcessor.scala

```diff
@@ -28,4 +32,16 @@ case class KafkaStreamProcessor[T](
       }
     }
   }
+
+  def retry[T](n: Int = retrycount)(fn: => T): T = {
```

Comment:
this might be a useful util function for others (also it's surprising this function doesn't exist elsewhere in the zipkin code?).